### PR TITLE
Update status badge doc

### PR DESCRIPTION
**Pull request by fatihdumanli**
status-badge

### CODE REVIEW

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

File: docs/pipelines/includes/get-status-badge.md [hunk 1]
```diff
@@ -19,9 +19,11 @@ To copy the status badge to your clipboard:
 
 1. In Azure Pipelines, go to the **Pipelines** page to view the list of pipelines. Select the pipeline you created in the previous section.
 
-2. On the upper right, select **Edit** to edit the pipeline build, and then select **Options**.
+2. On the upper right, click on three dots to expand the dropdown menu.
```
Comment (ShannonLeavitt) [inline docs/pipelines/includes/get-status-badge.md:22]:
```suggestion
2. In the upper-right corner, select the three dots to expand the dropdown menu.
```

File: docs/pipelines/includes/get-status-badge.md [hunk 2]
```diff
@@ -19,9 +19,11 @@ To copy the status badge to your clipboard:
...
 
-3. Copy the sample Markdown from the status badge section.
+3. Select **Status badge**
```
Comment (ShannonLeavitt) [inline docs/pipelines/includes/get-status-badge.md:24]:
```suggestion
3. Select **Status badge**.
```

**Comment by PRMerger19** [comment]
@ShannonLeavitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**